jira_agile_metrics/calculators/impediments.py

```diff
@@ -32,21 +32,23 @@ def run(self):
         ):
             logger.debug("Not calculating impediments data as no output files specified")
             return None
-        
-        backlog_column = self.settings['backlog_column']
-        done_column = self.settings['done_column']
 
```

Comment:
Again, could this change break configurations that rely on using these configuration options?